### PR TITLE
Restore the font cache on Travis.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Summary

followup to https://github.com/matplotlib/matplotlib/pull/9787#issuecomment-441441802
attn @jklymak 

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by jklymak** [comment]
I'll merge, or feel free to self-merge, when CI passes...  (don't thnk this needs two reviews)

File: lib/matplotlib/font_manager.py [hunk 1]
```diff
@@ -1278,12 +1278,15 @@ def _rebuild():
 try:
     fontManager = json_load(_fmcache)
 except Exception:
+    print("Rebuilding font cache.")
```
Comment (timhoffm) [inline lib/matplotlib/font_manager.py]: This code is executed at import time. I don't think we should pollute stdout with such messages. Maybe `_log.info()` or `_log.debug()`?

File: lib/matplotlib/font_manager.py [hunk 2]
```diff
@@ -1278,12 +1278,15 @@ def _rebuild():
...
     _rebuild()
 else:
     if getattr(fontManager, '_version', object()) != FontManager.__version__:
         _rebuild()
     else:
+        print("Reusing font cache.")
         _log.debug("Using fontManager instance from %s", _fmcache)
+json.dump(fontManager, sys.stdout, cls=JSONEncoder, indent=2)
```
Comment (timhoffm) [inline lib/matplotlib/font_manager.py]: Can you please add a comment how this is working? We're dumping the fontmanager to stdout at import time?!?

File: lib/matplotlib/font_manager.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/font_manager.py]: I'm just trying to debug the failure in Travis (this is a separate commit, in fact) as I can't repro the failure locally.

File: lib/matplotlib/font_manager.py (hunk 2 above)
Comment (anntzer) [reply lib/matplotlib/font_manager.py]: Same as above.

**Pull request merged**